sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1029,6 +1029,7 @@ public enum ServiceVersion
             V3_0 = 1,
             V3_1 = 2,
             V2022_05_01 = 3,
+            V2022_10_01_Preview = 4,
```

Comment:
So none of the new features have been exposed yet, right?

Reply:
Correct. I'll be exposing them one by one in subsequent PRs (unless there's a better way of doing it?)